external/fv3fit/fv3fit/pytorch/cyclegan/discriminator.py

```diff
@@ -89,7 +111,12 @@ def __init__(
                 stride=2,
             )
         )
+        if use_geographic_embedded_bias:
+            convs.append(
+                GeographicBias(channels=min_filters, nx=int(nx / 2), ny=int(ny / 2))
```

Comment:
This divide-by-2 for `nx` and `ny` comes from the `stride=2` for the first convolution layer? Any concern about not having the geographic bias on the native grid? (or put another way, could you do embedding without stride=2 convolution?)

Reply:
I'm not worried, unless I see poor performance in training. I'd be more worried about modifying the discriminator architecture by adding a non-strided convolution (if I did nonlinear embedding).

If I did first linearly embed into a `min_filters` space, then adding the bias before or after a strided convolution which follows would be identical, as it's a linear transformation. It's just that the bias would be over-defined and 75% of the information would be discarded.

Reply:
Okay, not sure I totally follow, but if performance is as expected, then I agree it's likely okay.